app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/check_your_details.scala.html

```diff
@@ -292,6 +292,17 @@ <h1 class="govuk-heading-l">@messages("cds.form.check-answers")</h1>
                     } else None
                 }
                 },
+                dateEstablished.flatMap { de => {
```

Comment:
Why do we have 2 of these now?

Reply:
Date Of establishment is displayed after the address for Row journey  but for UK journey doe is displayed before the address . Displaying the element conditionally according to the location

Reply:
Do we have unit tests that cover each scenario? Just to make sure that this content won't be displayed twice